server/controllers/generateReportController.test.ts

```diff
@@ -27,27 +27,30 @@ describe('generateReportController', () => {
 
```

Comment:
Can we add? :-
```
    request.session.successMessage = 'Download successful'
    request.session.downloadUrl = '/download-url'
```
So that we do not asset undefined on lines 32 & 33